app/assets/javascripts/modules/MultiTableFilter.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * # Multiple table filter
+ *
+ * Filters across one or more tables, and can also hide the elements associated with a table.
+ *
+ * Requires an element to have a data-dough-component="MultiTableFilter" attribute and an ID.
+ *
+ * Makes use of jquery.fastLiveFilter.
+ * See test fixture for sample markup - /spec/js/fixtures/MultiTableFilter.html
+ */
+
+define(['jquery', 'jqueryFastLiveFilter', 'DoughBaseComponent'],
+       function($, jqueryFastLiveFilter, DoughBaseComponent) {
+  'use strict';
+
+  var MultiTableFilterProto,
+      defaultConfig = {
+        filterTargetSelector: '.js-filterable',
```

Comment:
Sorry one other thing – we've typically used `data-dough` style attributes for this so its obvious what these things are for when reading the markup. See how the [TabSelector](https://github.com/moneyadviceservice/dough/blob/master/assets/js/components/TabSelector.js#L21-L22) does it.